extensions/wikidata/src/org/openrefine/wikidata/commands/PreviewWikibaseSchemaCommand.java

```diff
@@ -81,14 +84,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
                 return;
             }
 
+            Manifest manifest = null;
+            String manifestJson = request.getParameter("manifest");
+            if (manifestJson != null) {
+                try {
+                    manifest = ManifestParser.parse(manifestJson);
+                } catch (ManifestException e) {
+                    respondError(response, "Wikibase manifest could not be parsed.");
```

Comment:
Maybe I should append `e.getMessage()` to the message.

Reply:
Yes, that would be good. There's also `org.apache.commons.lang.exception.ExceptionUtils.getFullStackTrace(e)`, but hopefully the stacktrace won't be necessary because the range of errors here is limited.